### PR TITLE
Added WeightSize function for multihead_attention, multiply_constant and multiply_merge layers.

### PR DESCRIPTION
**Pull request by ayushsingh11**
This is in reference with issue #2590 .

### CODE REVIEW

File: src/mlpack/methods/ann/layer/multihead_attention.hpp [hunk 1]
```diff
@@ -120,6 +120,9 @@ class MultiheadAttention
                 const arma::Mat<eT>& error,
                 arma::Mat<eT>& gradient);
 
+  //! Get the size of the weights.
+  size_t WeightSize() const { return (4 * (embedDim + 1) * embedDim); }
```
Comment (zoq) [inline src/mlpack/methods/ann/layer/multihead_attention.hpp:124]: You can remove the surrounding `()`.

**Review by zoq — COMMENTED** [review]
The title mentioned `multiply_constant` and `multiply_merge` are you planning to include the layer part of the PR or open another PR?

**Comment by ayushsingh11** [comment]
Didn't completely get you, I added WeightSize() in multiply_constant and multiply_merge, was there anything else also to be done acc. to #2590 ?

**Comment by mrityunjay-tripathi** [comment]
Looks good to me.

**Review by zoq — APPROVED** [review]
Thanks, looks good to me.

**Comment by zoq** [comment]
Thanks for the contribution.

**Pull request merged**